css/index.css

```diff
@@ -77,9 +77,10 @@ label{
 
 
 /* Do an horizontal flip when you move the mouse over the flip box container */
-#aboutHackYU:hover .flip-card-inner > *{
-    background-color: rgba(241, 29, 40, 0.7);
-    color: rgba(241, 246, 249, 1);
+label:hover .flip-card-front{
```

Comment:
I think it might be better to make the rotation even smaller than 10 degrees... When I was looking at it, I thought that the card started flipping and then my computer froze and the card didn't continue flipping for some reason. Only then did I realize that I was supposed to click. To avoid this, it might be better to have it rotate even less. After testing it, I was thinking maybe even just 1 degree or .5 degrees. It could be, though, that you're trying to make the hover have a different effect, in which case you should just stick with what you have if you like it this way.